### ISSUE TITLE
A question about replay

### ISSUE THREAD

**Issue by KID0031** [histmeisah/Large-Language-Models-play-StarCraftII#13]
Hello! When I try to watch the replay on StarCraft 2, it says "Unable to open map", I tried copying all the maps into a folder but it doesn't work, do you know any way to watch that right now? Thanks a lot : )

The folders are like this:
![image](https://github.com/histmeisah/Large-Language-Models-play-StarCraftII/assets/51467434/43e42ba6-a6d0-406c-a025-cdb5997f814c)
![image](https://github.com/histmeisah/Large-Language-Models-play-StarCraftII/assets/51467434/b1c77c7d-5fa3-409f-bb1f-b367a9e9c904)

**Comment by histmeisah** [comment]
Yeah, this problem may be caused by Blizzard update or python-sc2 it self(when running agent vs agent). 

Now this company merely use editor very well, and only one staff maintain starcraft2 and heros of storm. 
Every update has caused several replays can open, and it is so common. 
So i have no idea to solve this problem unless BLZ recover starcraft2 official update.
